usafacts/delphi_usafacts/pull.py

```diff
@@ -52,6 +55,13 @@ def pull_usafacts_data(base_url: str, metric: str) -> pd.DataFrame:
     """
     # Read data
     df = pd.read_csv(base_url.format(metric=metric))
+    date_cols = [i for i in df.columns if i.startswith("2")]
```

Comment:
??? why `if i.startswith("2")`

Reply:
The county names are like 
```
countyFIPS            County Name State  StateFIPS  2020-01-22  2020-01-23  ...  2021-06-16  2021-06-17  2021-06-18  2021-06-19  2021-06-20  2021-06-21
```

So the dates start with 2. To be a bit more rigorous I could regex or try to date parse them, or just add a comment explaining.